update readme

robinsenior opened the pull request:
- remove references to demo app.
- add SPM docs

raulriera reviewed (approved):
I assume you meant to delete the project file?

robinsenior commented:
@raulriera I didn't actually, good catch. I've reverted it, but also have no idea how that would've happened – I just checked my shell history and there's no reference to deleting it.

Was there a reason we deleted the demo project? My intern was hoping to learn from it.

raulriera commented:
It didn’t work with SPM if I remember correctly.  And it was also getting included 

But now that we can hide resources in the packages. We can bring it back and not include it
